RadixWallet/Features/AccountRecoveryScan/Children/AccountRecoveryScanInProgress/AccountRecoveryScanInProgress.swift

```diff
@@ -69,7 +69,7 @@ public struct AccountRecoveryScanInProgress: Sendable, FeatureReducer {
 			active: IdentifiedArrayOf<Profile.Network.Account>,
 			inactive: IdentifiedArrayOf<Profile.Network.Account>
 		)
-		case failedToDerivePublicKey
+		case failed
```

Comment:
Why this change?

Reply:
Because it is not the only way to fail, we have **four** different failures:
* failed to calculate derivation paths
* failed to derive public keys
* failed to create account
* failed to scan network (either bad response or failed to parse JSON) using GW endpoint

But currently we do not act on them... so we can use one shared error for all cases, and in the future if we need to disambiguate we will do so. 

Agreed?

Reply:
Got it